Dockerfile

```diff
@@ -26,6 +26,8 @@ EXPOSE 8000
 ENV LONGHORN_MANAGER_IP http://localhost:9500
 ENV LONGHORN_UI_PORT 8000
 
+RUN mkdir -p /var/config/ && touch /run/nginx.pid && chown -R 499 /var/config /run/nginx.pid
```

Comment:
Why we need /run/nginx.pid? should it be in /var/run?

Reply:
Because it would show `[emerg] open() "/run/nginx.pid" failed (13: Permission denied)` without creating `/run/nginx.pid` and changing the owner. It will still happen when specifying `/var/run/nginx.pid`.
Specify the pid file location to `/var/config/nginx.pid` instead.

Reply:
I see. Why I was asking is if you install nginx package, you will notice the default path for pid is /var/run/ngnix.pid. 

```shell
af432ea928fc:/ # cat /etc/nginx/nginx.conf | grep -i pid
#pid        /var/run/nginx.pid;
```

What you can do is `mkdir -p /var/run` and change the owner instead of changing the path of pid at runtime.  

```shell
RUN mkdir -p /var/config/ && mkdir -p /var/run && chown -R 499 /var/{config,run}
```

Reply:
After some testings, I found the default path of nginx pid file is `/run/nginx.pid` (although the config file shows `#pid /var/run/nginx.pid;`) and `/var/run` is a symbolic link to `/run` (I am not sure if we could remove `/var/run` and re-create it as a new directory to save nginx pid file)
So I modified it to create pid file first in `/var/run` and change its owner to `nginx` (499)